src/main/resources/test_files/runtime/FunctionObjectIntersect.iq

```diff
@@ -0,0 +1,2 @@
+(:JIQS: ShouldRun; Output="( { "a" : [ abc, 2 ], "b" : [ 2, ab ], "c" : [ [ 1, 2 ], 5e+1 ] } )" :)
```

Comment:
Just a side note: in the long term, the syntax for expected results may need to be improved. Here you can see that this is a quoted string containing quotes. No need to address this now though if it works as is, no worries.